index.html

```diff
@@ -22,18 +22,23 @@
 			  color: #fff;
 			}
 			.demo-layout-transparent{
-				background: url('https://getmdl.io/assets/demos/transparent.jpg') center / cover;
+				background: white;
 			}
 			.demo-layout-transparent .mdl-layout__header,
 			.demo-layout-transparent .mdl-layout__drawer-button {
 				color: white;
 			}
+			.mine-size{
+				width: 985px;

```

Comment:
Para maging malawak 'yong Card. Iyon lang. For later adjustments para sa mobile.